packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -307,7 +307,7 @@ function PageRoot({ children }: PageRootProps) {
   return (
     <Container>
       <Stack
-        data-testid="page-root"
+        data-test-id="page-root"
```

Comment:
this one really tripped me up :D

Reply:
The reason it's called that way is so we can also use it for `getByTestId`: https://testing-library.com/docs/queries/bytestid/#api

Reply:
yeah, there another in another file that was `data-test-id` so i was trying that first, i guess we can use `data-testid` everywhere